zos_subsystems/ims/ims_provisioning-wazi/vars_wazi/ims-dbdc.yml

```diff
@@ -0,0 +1,190 @@
+TCPIP_PROFILE_TMP_MEMBER: IMSPROF
+# ---------------------------------------------------------------------------- #
+STEPLIB: DFSF10.SDFSRESL
+
+# The API key to use for artifactory repositories
+api_key: ''
+
+ims_function: create
+
+# The SSID for the IMS to be provisioned
+DFS_IMS_SSID: IMWZ
+
+# The user ID to use for IMS related provisioning steps
+DFS_IMS_USERID: '{{ TARGET_USERNAME | upper }}'
+
+# ---------------------------------------------------------------------------- #
+# The first HLQ to be used to store IMS related datasets
+# these include procs, OLDS, WADS, PROCLIB, JOBS, and application
+# specific datasets
+DFS_AUTH_LIB_HLQ1: IMSTESTL
+# The second HLQ to be used to store IMS related datasets
+# these include procs, OLDS, WADS, PROCLIB, JOBS, and application
+# specific datasets
+DFS_AUTH_LIB_HLQ2: IMWZ
+# ---------------------------------------------------------------------------- #
+# The first HLQ for system datasets that will be used
+# by IMS during provisioning. These should exist prior to
+# running playbooks
+DFS_AUTH_LIB_SYSHLQ1: DFSF10
+# The second HLQ for system datasets that will be used
+# by IMS during provisioning. These should exist prior to
+# running playbooks
+DFS_AUTH_LIB_SYSHLQ2:
+
+DFS_AUTH_LIB_SYSHLQ:
+# ---------------------------------------------------------------------------- #
+
+# ---------------------------------------------------------------------------- #
+#                            job related settings                           #
+# ---------------------------------------------------------------------------- #
+JOB_CLASS: A
+MSG_CLASS: H
+
+# ---------------------------------------------------------------------------- #
+#  Common Services related settings                                            #
+# ---------------------------------------------------------------------------- #
+OM_CONFIG: false
+
+# ---------------------------------------------------------------------------- #
+#                            storage related settings                           #
+# ---------------------------------------------------------------------------- #
+
+# Are the volumes we want to use managed by the storage management subsystem (SMS)?
+#DFS_SMS_ENVIRONMENT: true 
+DFS_SMS_ENVIRONMENT: false
+
+# ------------------- Storage Management Subsystem settings ------------------ #
+# -------------------------- used when SMS is in use ------------------------- #
+
+# If the volumes are managed by SMS, what SMS storage class should be used?
+# DFS_SMS_CLASS: SCNOSMS       # SCNOSMS does not work as expected
+DFS_SMS_CLASS: SCIMS
+# If the volumes are managed by SMS, what SMS management class should be used?
+DFS_SMS_MCLASS: NONE
+
+# -------------------------- volume related settings ------------------------- #
+# ------------------------ for when SMS is not in use ------------------------ #
+
+# first volume to use to store IMS datasets
+DFS_DS_VOLUME1: A5USR1
```

Comment:
Need to find out how to get this value for each ADCD release

Reply:
I have code to set this variable at run-time. Will check in the change after we finish this PR.